doc/governance/MAINTAINERS.md

```diff
@@ -0,0 +1,24 @@
+# Maintainers
+
+This document lists the Maintainers of the Project. Maintainers may be added once approved by the existing maintainers as
+described in the [Governance document](./GOVERNANCE.md).
+By adding your name to this list you are agreeing to abide by
+the Project governance documents and to abide by all of the Organization's polices, including
+the [code of conduct](https://github.com/vispy/vispy/blob/main/CODE_OF_CONDUCT.md),
+[trademark policy](../org/TRADEMARKS.md),
+and [antitrust policy](../org/ANTITRUST.md). If you are participating because of your affiliation with another
+organization (designated below), you represent that you have the authority to bind that organization to these policies.
+
+| **NAME**        | **Organization** |
+|-----------------|------------------|
+| Lorenzo Gaisfas | N/A              |
+| David Hoese     | N/A              |
+| Almar Klein     | N/A              |
+| Eric Larson     | N/A              |
+| Kai Mühlbauer   | N/A              |
+| Cyrille Rossant | N/A              |
+| Nicolas Rougier | N/A              |
```

Comment:
@brisvag @almarklein @larsoner @kmuehlbauer @rossant @rougier I've listed all of you as maintainers to the library. If you think I've missed someone let me know. If you'd like to not be on this list, also let me know.

As you can read in the GOVERNANCE.md document, being a maintainer now doesn't mean anything different than it did before. The general idea with this governance structure is that it is all about "good faith" consensus. We can do things to this project if we think it goes along with the other maintainers' opinions/desires. The only thing that *requires* explicit consensus from a certain amount of maintainers are changes to the governance document or if anything is disagreed with.

Reply:
You can remove me at this point, I'm not sure my vision of the project is clear enough anymore to make important decisions :)

Reply:
Thanks @larsoner. I've removed you from the maintainers list. I'm sure we'll still bug you when we can.